.storybook/themes/news.scss

```diff
@@ -1177,8 +1180,25 @@
 				video-player-inline-video: (
 					gap: 0,
 				),
-				links-bar: (
-					gap: var(--global-spacing-2),
+				xl-manual-promo: (
+					grid-template-rows: 1fr,
```

Comment:
This is redundant unless we are using a Grid - I don't see any Grid used

Reply:
Yep, missed removing this.